packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -381,8 +382,14 @@ function useFocusContainment(scopeRef: RefObject<Element[] | null>, contain?: bo
         cancelAnimationFrame(raf.current);
       }
       raf.current = requestAnimationFrame(() => {
+        // Patches infinite focus coersion loop for Android Talkback where the user isn't able to move the virtual cursor
+        // if within a containing focus scope. Bug filed against Chrome: https://issuetracker.google.com/issues/384844019.
+        // Note that this means focus can leave focus containing modals due to this, but it is isolated to Chrome Talkback.
+        let modality = getInteractionModality();
+        let shouldSkipFocusRestore = (modality === 'virtual' || modality === null) && isAndroid() && isChrome();
+
         // Use document.activeElement instead of e.relatedTarget so we can tell if user clicked into iframe
-        if (ownerDocument.activeElement && shouldContainFocus(scopeRef) && !isElementInChildScope(ownerDocument.activeElement, scopeRef)) {
+        if (!shouldSkipFocusRestore && ownerDocument.activeElement && shouldContainFocus(scopeRef) && !isElementInChildScope(ownerDocument.activeElement, scopeRef)) {
```

Comment:
As stated here, this is a temporary patch. Note that this will break use cases where focus is truly lost to the body such [as this story on main](https://reactspectrum.blob.core.windows.net/reactspectrum/50c7ada5d1880a174b6b6d3f43e8d90ee9bd4ad8/storybook/index.html?path=/story/focusscope--focusable-first-in-scope&providerSwitcher-express=false) where due to the dialog's content changing, the previously focused button disappears. We would usually recoerce focus back into the dialog but at the moment we cannot differentiate between that  case and if the user is simply navigating between elements in the FocusScope in Chrome Talkback